src/styles/mixins.scss

```diff
@@ -0,0 +1,78 @@
+@mixin hover ($property, $toValue) {
+  transition: #{$property} $effect-duration;
+  &:hover {
+    #{$property}: #{$toValue};
+  }
+}
+
+@mixin colorHover ($property, $toValue) {
+  transition: #{$property} $effect-duration;
+  &:hover {
+    #{$property}: #{$toValue};
+  }
+}
+
+@mixin largeAndChangeColorOnHover {
+  &:hover {
+    transform: scale(1.2);
+    z-index: 1;
+    transition: transform $effect-duration,
+      color $effect-duration;
```

Comment:
`transition` should be applied to the element, not just `:hover` selector. Also, in this mixin color change is missing.

Actually, I think this mixin is redundant and easier just manually set when you need to specify for a few properties as in most cases they are different